changelog/unreleased/kong/11624-2.yml

```diff
@@ -0,0 +1,7 @@
+message: Bump lua-kong-nginx-module from 0.6.0 to 0.7.1
+type: dependency
+scope: Core
+prs:
```

Comment:
line 4~7 can be removed from new changelog doc including` prs` and `jiras`.

Reply:
Also could you change the the filename to `request_id.yml` instead? We are not discouraging the use of PR# as filename as email states.

Reply:
done, thanks